src/lib/input/input-container.scss

```diff
@@ -159,6 +160,24 @@ md-input-container {
   }
 }
 
+// Used to hide the placeholder overflow on IE, since IE doesn't take transform into account when
+// determining overflow.
+.md-input-placeholder-wrapper {
+  position: absolute;
+  left: 0;
+  top: -1em;
+  width: 100%;
+  padding-top: 1em;
+  overflow: hidden;
+  pointer-events: none;  // We shouldn't catch mouse events (let them through).
+
+  // Keeps the element height since the placeholder text is `position: absolute`.
+  &::after {
+    content: '\\00a0';
```

Comment:
Hmm, not sure about it. In this case it's useful, but in others where there may be content in the element, it may end up pushing it.